File: JAVA/JavaOdev2/RASTGELEKISIURET/src/Kisiler/Kisi.java
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 01.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */

package Kisiler;


import java.io.*;
import java.util.Random;

public class Kisi 
{
    
   private String isimSoyİsim ;
   public String isimGetir()
   {
       return isimSoyİsim;
   }
   public void isimAta() throws FileNotFoundException,IOException
   {
       Random rand = new Random();
       FileReader dosya = new FileReader("random_isimler.txt");
       BufferedReader br = new BufferedReader(dosya);
       int index = rand.nextInt(3000);
       if(index == 0)
            index = 254;
       int i = 0;
       String str;
       while((str = br.readLine()) != null )
       {
           if(i == index)
           {
               isimSoyİsim = str;
               br.close();
               break;
           }
           i++;
       }

        
   }
  
   private int yas;
   public int yasGetir()
   {
       return yas;
   }
   public void yasAta()
   {
       Random rand = new Random();
       yas = rand.nextInt(100);
   }
   
   
   public KimlikNo kimlik = new KimlikNo();
   
   public Telefon telefon = new Telefon();
    
    
    
}

File: C/Odev3-PDP/src/Kisi.c
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 22.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */


#include "Kisi.h" 
#include "stdio.h"
#include "stdlib.h"
#include "time.h"
#include "string.h"

void isimAta(Kisi ki)
{
	
	FILE* dosya ;
	
	
	dosya = fopen("random_isimler.txt" , "r");
	
	int index = rand() % 3000;
       
	   if(index == 0)
	   {
		  index = 254;
	   }
	
	int i = 0;
	char *str,*str2;
	
	char c[50];
	
	while(!feof(dosya))
	{
	
		str = fgets(c,50,dosya);
		
		
		if(i == index)
		{
			
				str2 = strtok(c,"\n");
				strcpy(ki->isimSoyisim,str2);
				
			
			
			fclose(dosya);
			
			
			
			break;
		}
		
		i++;
	
	}
	
	
}

void yasAta(Kisi ka)
{
	
	int _yas = rand() % 100 ;
	ka->yas = _yas ;
	
}

Kisi kisiOlustur()
{
	Kisi k ;
	k = (Kisi)malloc(sizeof(struct KISI));
	isimAta(k);
	yasAta(k);
	k->telefon = telefonUret();
	k->kimlik = tcNoUret();
}

	
           
	